apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnection.cs

```diff
@@ -90,6 +90,21 @@ public override string ConnectionString
         [Category("Data")]
         public string SpannerInstance => Builder.SpannerInstance;
 
+        /// <summary>
```

Comment:
We should probably ask the Spanner team about this. There's too much I don't know.

If we expose it at all though, I'd be tempted to make it a parameter on the method instead of having it as a property. (With a suitable default if null is passed in.)